bin/env_variables_all.txt

```diff
@@ -17,6 +17,9 @@ CCD_BEFTA_MASTER_CASEWORKER_PWD=Pa55word11
 CCD_BEFTA_MASTER_SOLICITOR_1_PWD=Pa55word11
 CCD_BEFTA_MASTER_SOLICITOR_2_PWD=Pa55word11
 CCD_BEFTA_MASTER_SOLICITOR_3_PWD=Pa55word11
+CCD_BEFTA_MASTER_SOLICITOR_4_PWD=Pa55word11
```

Comment:
This is used by MCA and is defined in MCA's own env_variables_all.txt [here](https://github.com/hmcts/aac-manage-case-assignment/blob/master/aca-docker/bin/env_variables_all.txt)

Reply:
If this is not required here then it can be removed. All other passwords are present so made sense to add this one

Reply:
Have been using this PR tro test locally and all works ok, so happy to merge as is